public/scripts/ulid.min.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Minified by jsDelivr using Terser v5.19.2.
+ * Original file: /npm/ulid@2.3.0/dist/index.umd.js
```

Comment:
@pkong-ds the file already contains version number

Reply:
LOL, didn't expect a `min.js` to have spaces & indents for readability.

Coz normally dev is not expected to `open a min.js file and read it`.

Anyway, think this is good enough